Packs/Reco/ReleaseNotes/1_0_1.md

```diff
@@ -0,0 +1,4 @@
+#### Integrations
```

Comment:
❯ markdownlint 1_0_1.md
1_0_1.md:1 MD022/blanks-around-headings/blanks-around-headers Headings should be surrounded by blank lines [Expected: 1; Actual: 0; Below] [Context: "#### Integrations"]
1_0_1.md:1 MD041/first-line-heading/first-line-h1 First line in a file should be a top-level heading [Context: "#### Integrations"]
1_0_1.md:2 MD022/blanks-around-headings/blanks-around-headers Headings should be surrounded by blank lines [Expected: 1; Actual: 0; Above] [Context: "##### Reco"]
1_0_1.md:2 MD022/blanks-around-headings/blanks-around-headers Headings should be surrounded by blank lines [Expected: 1; Actual: 0; Below] [Context: "##### Reco"]
1_0_1.md:3 MD032/blanks-around-lists Lists should be surrounded by blank lines [Context: "- Fetch Incidents from Reco"]